nifi-nar-bundles/nifi-standard-bundle/pom.xml

```diff
@@ -304,6 +304,11 @@
                 <artifactId>guava</artifactId>
                 <version>31.0.1-jre</version>
             </dependency>
+            <dependency>
+                <groupId>com.networknt</groupId>
+                <artifactId>json-schema-validator</artifactId>
+            <version>1.0.72</version>
```

Comment:
Looks like json-schema-validator is up to version 1.0.73 as seen [here](https://github.com/networknt/json-schema-validator), should I use the latest version?

Reply:
Yes, that would be helpful.